### DISCUSSION TITLE
How to 'unlink' databases from the 'Select Database' drop down

### DISCUSSION THREAD

**Discussion by ghost** [Sequel-Ace/Sequel-Ace#1700] [Q&A]
I have somehow 'linked' other databases to one of my other databases. Not sure how else to describe it! When I click on the dropdown above 'Select Database' it is showing 3 databases. But on other databases only the active database shows.

I deleted the saved favourite and re-added it but the 'linked' databases still show up in that dropdown. Any idea how to remove them without actually deleting the database from the server?

<img width="221" alt="Screenshot 2023-02-06 at 12 28 45" src="https://user-images.githubusercontent.com/4290631/216972281-b27a1254-6f52-4146-b7ba-b60291c2cc5b.png">

<img width="201" alt="Screenshot 2023-02-06 at 12 28 30" src="https://user-images.githubusercontent.com/4290631/216972259-12d89575-92d8-447d-815d-4e0d867c8fe3.png">

In the screenshot above, I do not want the _anz or _usa databases to be linked to the current _uk database.

**Comment by samuelbradshaw** [comment]
My understanding is that this dropdown always shows all databases on the currently connected server. They aren't linked, other than being on the same server.

When you say "on other databases only the active database shows," do those other databases have multiple databases on the server?

**Reply by Jason-Morcos** [reply]
@samuelbradshaw Is correct. It shows all the databases on that server that the account you're using has access to. So another possibility is that the other accounts for your other connections are scoped down to only have access to some databases via permissions. Sequel Ace isn't doing anything magic about linking connections, it's just allowing you to see what you can access on the server itself.

**Reply by ghost** [reply]
> My understanding is that this dropdown always shows all databases on the currently connected server. They aren't linked, other than being on the same server.
> 
> When you say "on other databases only the active database shows," do those other databases have multiple databases on the server?

Ah, now you've said that... I'll check which user details I'm using as I may have used a 'shared access' user account.

Thanks for the tip!
